linera-execution/src/key_value_store_view.rs

```diff
@@ -0,0 +1,399 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//! We implement a `KeyValueStoreView` implements View and the function of `KeyValueStore`.
```

Comment:
This doesn't belong here

Reply:
Also it could be called `SmallKeyValueStoreView` if we want to keep the old implementation around